common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/requests/resetpassword/ResetPasswordStartRequest.kt

```diff
@@ -53,6 +53,7 @@ data class ResetPasswordStartRequest private constructor(
             headers: Map<String, String?>
         ): ResetPasswordStartRequest {
             // Check for empty Strings and empty Maps
+            ArgUtils.validateNonNullArg(username, "username")
```

Comment:
why was this removed?

Reply:
Not sure about the reason but I would prefer to keep the checking here.